Topics/math-fundamentals.md

```diff
@@ -0,0 +1,172 @@
+# Math Fundamentals
+
+# What you will learn 
+In this guide, you'll learn what is the fundamental mathematical knowledge you need to have in order
+to build insightful Data Science projects and build machine learning systems upon them.
+
+You'll also learn that not so much is needed to start to apply yourself to real-world data problems!
+
+## Prerequisites
+The previous Paradiso guides.
+
+## Time to complete
+10 minutes.
+
+# Index
+- [Motivation](#Motivation)
+- [Courses](#Courses)
+  - [Calculus](#Calculus)
+  - [Linear Algebra and Matrix Algebra](#Linear-Algebra)
+  - [Optimization](#Optimization)
+- [Ask Questions](#Ask-Questions)
+- [Conclusions](#Conclusions)
+
+Let's dive right in!
+ 
+### Motivation
+
+Before anything else, watch [this](https://www.youtube.com/watch?v=VIbjHIGMjQM
+) and [this](https://www.youtube.com/watch?v=8CX-Q0gtSp8) awesome Ted Talks :-)
+
+**The entire field of Data Science is based on mathematics and statistics.**
+
+This phrase shouldn't frighten you, though, because as you'll find out, you probably have much of the knowledge you need to start experimenting!
+
+**In fact:** 
+
+If you've done a second-degree school (you know matrices and derivatives, for example), you probably just need to refresh yourself on the things you already know and go deeper into some things you may not have seen at school. 
+
+In general, if you have a scientific degree (engineering, computer science, physics, etc.), it will probably be just a review, as it will be the Calculus, Linear Algebra, and Matrix Algebra courses are suggested within this guide.
+
+Whatever your starting level, however, it is still recommended to go through all the resources organized here, _to make sure you do not have "fundamental knowledge holes"._
+
+Remember to read and try to implement some ideas of the [**Virgilio's Learning Strategy - Learning to Learn**](teaching.md)
```

Comment:
👍 

I can also recommend [Coursera's course: Learning how to learn](https://www.coursera.org/learn/learning-how-to-learn/)

Reply:
Very good course, Ill put in the "learn to learn" guide